src/sentry/static/sentry/app/components/emptyStateWarning.jsx

```diff
@@ -47,14 +48,10 @@ const EmptyStreamWrapper = styled.div`
       margin-bottom: 0;
     }
   }
-`;
 
-const Icon = styled.div`
-  display: block;
-  font-size: 54px;
-  color: ${p => p.theme.gray2};
-  margin-bottom: 20px;
-  opacity: 0.45;
+  svg {
```

Comment:
It would probably be slightly more correct to use `${HeroIcon}` as the selector here